cmd/mdmctl/config.go

```diff
@@ -80,21 +81,35 @@ func setCmd(cfg *ClientConfig, args []string) error {
 	return SaveClientConfig(cfg)
 }
 
-func clientConfigPath() (string, error) {
+func clientConfigDirPath() (string, error) {
 	usr, err := user.Current()
 	if err != nil {
 		return "", err
 	}
-	path := usr.HomeDir + "/.micromdm/default.json"
-	return path, nil
+	return path.Join(usr.HomeDir, ".micromdm"), nil
+}
+
+func clientConfigPath() (string, error) {
+	dir, err := clientConfigDirPath()
+	return path.Join(dir, "default.json"), err
```

Comment:
Separate func to get just the directory (to make it). Alternatively could just return an []string and have the last user path.Join() them. I don't like post-parsing path strings.